README-ja-JP.md

```diff
@@ -0,0 +1,197 @@
+
+- 評価結果は [OpenCompass 20230706](https://github.com/internLM/OpenCompass/) (*印のあるデータは原著論文からの引用を意味する)から取得したもので、評価設定は [OpenCompass](https://github.com/internLM/OpenCompass/) が提供する設定ファイルに記載されています。
+- 評価データは、[OpenCompass](https://github.com/internLM/OpenCompass/) のバージョンアップにより数値的な差異が生じる可能性がありますので、[OpenCompass](https://github.com/internLM/OpenCompass/) の最新の評価結果をご参照ください。
+
+### Model Zoo
+
+InternLM 7B と InternLM 7B チャットは、InternLM を使って訓練され、オープンソース化されています。モデルの重みは 2 つのフォーマットで提供されています。Transformers フォーマットを使ってモデルをロードするだけでなく、InternLM を使って直接重みをロードして、さらに事前トレーニングや人間の好みアライメントトレーニングを行うこともできます。
+
+| モデル                         | InternLM フォーマット Weight ダウンロードリンク                                                                                                                 | Transformers フォーマット Weight ダウンロードリンク                                         |
+| ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| **InternLM 7B**         | [![Open in OpenXLab](https://cdn-static.openxlab.org.cn/header/openxlab_models.svg)](https://openxlab.org.cn/models/detail/OpenLMLab/InternLM-7b)         | [🤗internlm/intern-7b](https://huggingface.co/internlm/internlm-7b)                 |
+| **InternLM Chat 7B**    | [![Open in OpenXLab](https://cdn-static.openxlab.org.cn/header/openxlab_models.svg)](https://openxlab.org.cn/models/detail/OpenLMLab/InternLM-chat-7b)    | [🤗internlm/intern-chat-7b](https://huggingface.co/internlm/internlm-chat-7b)       |
+| **InternLM Chat 7B 8k** | [![Open in OpenXLab](https://cdn-static.openxlab.org.cn/header/openxlab_models.svg)](https://openxlab.org.cn/models/detail/OpenLMLab/InternLM-chat-7b-8k) | [🤗internlm/intern-chat-7b-8k](https://huggingface.co/internlm/internlm-chat-7b-8k) |
+
+**制限事項:** 学習過程におけるモデルの安全性を確保し、倫理的・法的要件に準拠したテキストを生成するようモデルに促す努力を行ってきたが、モデルのサイズと確率的生成パラダイムのため、モデルは依然として予期せぬ出力を生成する可能性がある。例えば、生成された回答には偏見や差別、その他の有害な内容が含まれている可能性があります。そのような内容を伝播しないでください。有害な情報の伝播によって生じるいかなる結果に対しても、私たちは責任を負いません。
+
+### Transformers からのインポート
+
+Transformers を使用して InternLM 7B チャットモデルをロードするには、以下のコードを使用します:
+
+```python
+>>> from transformers import AutoTokenizer, AutoModelForCausalLM
+>>> tokenizer = AutoTokenizer.from_pretrained("internlm/internlm-chat-7b", trust_remote_code=True)
+>>> model = AutoModelForCausalLM.from_pretrained("internlm/internlm-chat-7b", trust_remote_code=True).cuda()
+>>> model = model.eval()
+>>> response, history = model.chat(tokenizer, "hello", history=[])
+>>> print(response)
+こんにちは！ 今日はどのようなご用件でしょうか？
+>>> response, history = model.chat(tokenizer, "please provide three suggestions about time management", history=history)
+>>> print(response)
+もちろん、効果的な時間管理のための3つのヒントはここにあります:
+
+1. 重要度と緊急度に基づいてタスクの優先順位をつける: すべてのタスクをリストアップし、"重要かつ緊急"、"重要だが緊急ではない"、"重要ではないが緊急"に分類する。最初のカテゴリーのタスクを完了させることに集中してから、他のカテゴリーに移る。
```

Comment:
Hi @eltociear ，
Just want to make sure that are these contents produced by the InternLM-Chat or simply translated from the origin documents?

Reply:
@ZwwWayne Sorry, this is simply my translation of the original text.